docs/zh_cn/datasets/det.md

```diff
@@ -53,13 +42,129 @@
 
 ## 准备步骤
 
+## CTW1500
+
+- 第一步: 运行以下命令, 从 [github](https://github.com/Yuliang-Liu/Curve-Text-Detector)下载`train_images.zip`, `test_images.zip`, `train_labels.zip`, `test_labels.zip`等四个文件。
+
```

Comment:
Between English and Chinese words, we usually add a `space blank` for better viewing.